great_expectations/util.py

```diff
@@ -139,8 +139,8 @@ def underscore(word: str) -> str:
     return word.lower()
 
 
-def hyphen(input: str):
-    return input.replace("_", "-")
+def hyphen(txt: str):
+    return txt.replace("_", "-")
```

Comment:
love the rename, but wanted to confirm whether this method is only called from within GE (otherwise it could be a breaking change)

Reply:
Only from GE -- rename is due to PyCharm warning (we were overriding a reserved word).  Thanks, @Shinnnyshinshin !